changelog/v0.30.0/update-controller-runtime.yaml

```diff
@@ -0,0 +1,5 @@
+changelog:
+  - type: BREAKING_CHANGE
```

Comment:
could you also list the DEPENDENCY_BUMPS, particularly k8s.io/api and controller-runtime?

Reply:
Yeah, those seem reasonable.  All were a consequence of pulling in controller-runtime.